models/Pet.js

```diff
@@ -9,18 +9,15 @@ Pet.init(
             type: DataTypes.STRING,
             allowNull: false,
         },
-        // TODO: change datatypes?
         age: {
-            type: DataTypes.DECIMAL,
+            type: DataTypes.STRING,
             allowNull: false,
         },
-        // TODO: change datatypes
         gender: {
             type: DataTypes.STRING,
         },
-        // TODO: change datatypes
         weight: {
-            type: DataTypes.INTEGER
+            type: DataTypes.STRING,
```

Comment:
Please make 'INTEGER'

Reply:
Just like age, some pet like snake or hamster are really light, not even 1Lbs so INTEGER wont do it